src/main/java/com/eva/logic/commands/AddCommentCommand.java

```diff
@@ -27,66 +34,76 @@
 
 public class AddCommentCommand extends CommentCommand {
 
-    public static final String COMMAND_WORD = "addcomment";
-
-    public static final String MISSING_PERSONTYPE_MESSAGE = "Need to specify applicant or staff \n"
-            + "e.g: 'addcomment s- ...' or 'addcomment a- ...'";
+    public static final String COMMAND_WORD = "addc";
 
     public static final String MESSAGE_ADDCOMMENT_USAGE = "Format for this command: \n"
-            + COMMAND_WORD + " INDEX <s-/a-> ti/TITLE d/DATE desc/DESCRIPTION";
+            + COMMAND_WORD + " INDEX c/ ti/TITLE d/DATE desc/DESCRIPTION";
+    public static final String MESSAGE_DUPLICATE_COMMENT = "Duplicate comment titles not allowed";
 
-    private String personType;
 
     /**
      * Creates an addcommentcommand object
      * @param index
      * @param commentPersonDescriptor
-     * @param personType
      */
-    public AddCommentCommand(Index index, CommentCommand.CommentPersonDescriptor commentPersonDescriptor,
-                             String personType) {
+    public AddCommentCommand(Index index, CommentCommand.CommentPersonDescriptor commentPersonDescriptor) {
         super(index, commentPersonDescriptor);
-        this.personType = personType;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        //for now is staff because we only working with staff for now
+        PanelState personType = model.getPanelState();
```

Comment:
Should change the name to panelState or currentPanel or something along those lines